test/functional/legacy/060_exists_and_has_functions_spec.lua

```diff
@@ -644,11 +648,10 @@ describe('exists() and has() functions', function()
       call TestHas()
 
       edit! test.out
-      set ff=unix
```

Comment:
I am not sure any more why I thought that. Maybe because Windows and Mac support is now gone. Should I put it back in just to be save?